src/plugins/bbsync/interfaces/HeartbeatInterface.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE interface SYSTEM "interface.dtd">
+<interface name="HeartbeatInterface" author="Mike Henning" year="2022">
+  <data>
+    <comment>
+      Interface to check on the availablity of the robotino
```

Comment:
this has nothing to do with the robotino (same for the comment on the `alive` field)